tests/unit/states/archive_test.py

```diff
@@ -72,15 +67,9 @@ def test_extracted_tar(self):
                                                    'file.file_exists': mock_false,
                                                    'file.makedirs': mock_true,
                                                    'cmd.run_all': mock_run}):
-                    if HAS_PWD:
-                        running_as = pwd.getpwuid(os.getuid()).pw_name
```

Comment:
I know that if i run that test on my Arch as me, it will fail, on arch, my username appears in the path. Guess that's an arch thing and not a fedora thing, meaning, it depends on the distro(if not running as root).